### DISCUSSION TITLE
Bibliography relocation in journal articles

### DISCUSSION THREAD

**Discussion by sammo3182** [quarto-dev/quarto-cli#1728] [Q&A] [answered]
Is there a way to relocate the bibliography before certain sections, such as the appendices? #712 is talked about in the context of book, but it does not work for articles. The `bookdown` ways don't work (why? both Pandoc, right?), i.e.,

```
::: {#refs}
:::
```

and 

```
<div id="refs"></div>
```

**Comment by rgaiacs** [comment, marked as answer]
Hi @sammo3182, the following minimal working example

```
---
title: "Minimal Working Example"
bibliography: references.bib
---

## Text

Awesome R package[@tidyverse].

# References {.unnumbered}

::: {#refs}
:::

## More Text
```

works for me when converting to HTML.

![Screenshot from 2022-08-08 10-31-48](https://user-images.githubusercontent.com/1506457/183326649-c2dab469-4b6a-479c-86e5-ba38f4e763d1.png)

**Reply by sammo3182** [reply]
Thanks a lot @rgaiacs ! I tried that, not work but then I found the set works only when I changed the `qmd` file name. No idea why, but......
